spring/src/main/java/io/seata/spring/remoting/parser/RemotingFactoryBeanParser.java

```diff
@@ -81,4 +81,8 @@ public RemotingDesc getServiceDesc(Object bean, String beanName) throws Framewor
     public short getProtocol() {
         return 0;
     }
+
+    public static void setApplicationContext(ApplicationContext applicationContext) {
```

Comment:
Remove this method and change the member variables as follows, which I think is more appropriate
` public static final ApplicationContext APPLICATION_CONTEXT = ObjectHolder.INSTANCE.getObject(ApplicationContext.class);`

Reply:
I think the RemotingFactoryBeanParser is load earlier, so can't get ApplicationContext this way

Reply:
> I think the RemotingFactoryBeanParser is load earlier, so can't get ApplicationContext this way

GlobalTransactionScanner should be loaded before RemotingFactoryBeanParser.